Dockerfile

```
@@ -93,7 +93,17 @@ ENV JAVA_TOOL_OPTIONS="-XX:+UseShenandoahGC \
     --enable-native-access=ALL-UNNAMED \
     --enable-preview"
 
-RUN apk add --no-cache su-exec libstdc++ libgcc libarchive && \
+RUN printf '%s\n' \
+    '@edge-main https://dl-cdn.alpinelinux.org/alpine/edge/main' \
+    '@edge-community https://dl-cdn.alpinelinux.org/alpine/edge/community' \
+    '@edge-testing https://dl-cdn.alpinelinux.org/alpine/edge/testing' \
+    >> /etc/apk/repositories
+
+RUN apk add --no-cache --upgrade \
```

Hadolint (2.14.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)